raiden/raidenwebui/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see http://editorconfig.org
```

Comment:
Is it needed? If it's needed for angular keep it. But it seems like something that does not need to be comitted.